tests/expectations/tests/float128.rs

```diff
@@ -0,0 +1,19 @@
+/* automatically generated by rust-bindgen */
+
+
+#![allow(non_snake_case)]
+
+
+#[repr(C)]
+#[derive(Debug, Copy)]
+pub struct A {
+    pub f: [u8; 16usize],
+}
+#[test]
+fn bindgen_test_layout_A() {
+    assert_eq!(::std::mem::size_of::<A>() , 16usize);
+    assert_eq!(::std::mem::align_of::<A>() , 16usize);
```

Comment:
I doubt a lot the alignment of `f128` is `16`. The same happens with complex types and `i128` and friends. Land the test commented with a FIXME and with empty expectations, since there's no way of getting a 128-bit aligned value in rust as of right now.

Also, note that layout tests now don't run by default in regen (probably should be changed? I don't know, makes most of the use cases faster), and you need to run `cargo test -p tests_expectations`.